cves/kernel/CVE-2012-6545.yml

```diff
@@ -401,37 +412,37 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
```

Comment:
While it's not permissions-related, it does highlight that the struct object should be securely initialized before use

Reply:
Perhaps.